R/groupwise_diff.R

```diff
@@ -0,0 +1,23 @@
+#' Groupwise Differenence (Fairness Metrics)
+#' @name groupwise_diff
+#'
+#' @param prediction The predictions of the learner.
+#' @param base_measure The base measures used to evaluate.
+#' @param data_task The data task for the fairness metric.
+#' @param ... Further arguments, currently ignored.
+#'
+#' @return Difference of base measues between binary protected groups.
+#' @export
```

Comment:
No export, data types. But for non-exported functions just drop change `#'` to `#`. This holds for all functions that are internally used in `MeasureFairness`.

Reply:
Fixed